.github/workflows/lint.yml

```diff
@@ -0,0 +1,12 @@
+name: lint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: make lint
```

Comment:
Any reason for not downloading and using the release binary from golangcli-lint? Trying to understand what the standard practice should be

Reply:
I lean toward a container image that can be shared with the rest of the org, but I wanted to open a self-contained PR to demonstrate utility first. Good question -- curious what opinions others have.